### ISSUE TITLE
Question generation

### ISSUE THREAD

**Issue by SRIKARHI** [artitw/text2text#1]
Does the model generates  questions only of "What" type.
every time we run a model with input sentence one question is generated. Next time for the same input different question is generated. Is there any way to generate all possible questions for a given sentence at one go.

**Comment by artitw** [comment]
No, “what” questions are just the most common. Try controlling the answer with the `[SEP]` token. It would take some time to properly generate an exhaustive list of questions for a given input. Will keep you updated
